packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -402,4 +451,170 @@ void main() {
 
     handle.dispose();
   });
+
+  testWidgets('Ensure canTapOnHeader is false by default',  (WidgetTester tester) async {
+    final expansionPanel = ExpansionPanel(
```

Comment:
I would expand this to actually use `pumpWidget` with the appropriate setup, since you want to test it with as practical a use case as possible and ideally none of the surrounding code and widgets causes any unintended changes to the behavior.

Reply:
You should also write a test for `ExpansionPanelRadio` as well to be thorough

Reply:
The effect of having `canTapOnHeader` default to `false` is tested as part of 'Do not toggle simple panel list by clicking on header when canTapOnHeader is false' test case by using `pumpWidget`. Therefore, I thought that a unit test to safeguard the default value would be enough.